src/applications/resources-and-support/style.scss

```diff
@@ -22,3 +22,9 @@
   border-bottom-left-radius: 5px;
   border-bottom-right-radius: 5px;
 }
+
+@media print {
+  #resources-search-bar-wrapper {
```

Comment:
Asking again if we can use a class for this. The specificity index for IDs is way too high and results in the use of !important down the road. Eventually we'll lint against this, but I'd like to avoid it all together so we don't have to spend unnecessary time cleaning it up when we can just set a good example now and use a class.

https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity
https://css-tricks.com/specifics-on-css-specificity/

Reply:
While I disagree that using an element ID  is always tech debt or that there is never a use case for them, I made this 
requested change because of the insight you've shared into VSP's intention of removing element IDs and disallowing them from the project.